metrics/utils.go

```diff
@@ -16,11 +16,39 @@ limitations under the License.
 
 package metrics
 
-import "strconv"
+import (
+	"strconv"
+
+	"go.opencensus.io/tag"
+)
 
 // ResponseCodeClass converts an HTTP response code to a string representing its response code class.
 // E.g., The response code class is "5xx" for response code 503.
 func ResponseCodeClass(responseCode int) string {
 	// Get the hundred digit of the response code and concatenate "xx".
 	return strconv.Itoa(responseCode/100) + "xx"
 }
+
+// MaybeInsertInt conditionally insert the tag when cond is true.
+func MaybeInsertInt(key tag.Key, value int, cond bool) tag.Mutator {
```

Comment:
Do you mean `InsertIntTag`? I think it's important to clarify, since the package name doesn't indicate the context here (`metrics` vs. `tag`)

Reply:
good point! Fixing...